src/Autocomplete/doc/index.rst

```diff
@@ -171,6 +171,109 @@ After creating this class, use it in your form:
 
 Congratulations! Your ``EntityType`` is now Ajax-powered!
 
+.. _passing-extra-options-to-the-ajax-powered-autocomplete:
+
+Passing Extra Options to the Ajax-powered Autocomplete
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.13
+
+    The ability to pass extra options was added in LiveComponents 2.13.
+
+By default, when you pass any options while adding a field to a form type, they will be lost
+when the autocomplete field is rendered on an Ajax call. To partially avoid this limitation,
+the `extra_options` option was added. It can only hold a scalar values, but it covers most use cases.
+
+Considering the following example, when the form type is rendered for the first time, it will use the `query_builder` defined
+while adding a `food` field to the `FoodForm`. However, when the Ajax is used to fetch the results, on the consequent renders,
+the default `query_builder` will be used::
```

Comment:
Just a personal feeling (and my english is quite limited so take this with a pinch of salt :) )

And i think @javiereguiluz vision would be much usefull than mine there :) 

What i feel here is that those two paragraphs may be written with a tone more "neutral / factual" (not sure of the term) 

Something like 

```
Autocomplete field options are not kept from a request to another, so following renders ... 

To persists options betwen [..], you must use extra_options [..]

::warning
only scalar (string, booleans ....) 
```

And then the example. 

not sure if what i'm saying is clear 👼